### PR TITLE
#166789980 Setting Up Sequelize And Postgres

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
- Sets up sequelize in the application

#### Description of Task to be completed
- Setup `sequelize` and `postgres`
- Configure connection  through the `.sequelizerc` file
#### How should this be manually tested?
1. Run the following commands in terminal
```
git clone git@github.com:andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ch-setup-postgresql-166789980
npm install
```
2. Create a .env file and add the following lines
```
DATABASE_URL=postgres://username:password/5423/database_name
DATABASE_TEST=postgres://username:password/5423/database_testname
```
3. Start the application with `npm run start`

4. Use `npm run test` to run tests

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789980](https://www.pivotaltracker.com/n/projects/)
#### Screenshots (if appropriate)
N/A
#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
```
Comment (Hervera) [inline .sequelizerc]: Why are you still using ES5 in your codes?

Comment (kodek-sleuth) [reply .sequelizerc]: You need to be professional in your commenting. That statement should have been 'Use ES6'

**Comment by dmithamo** [comment]
Fix the header on your PR. Look at other PR’s if you need guidance

File: config/index.js [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+    use_env_variable: 'DATABASE_TEST',
+    dialect: 'postgres',
+    logging: false,
+  },
+  production: {
+    use_env_variable: 'DATABASE_URL',
+    dialect: 'postgres',
+    logging: false
+  },
+};
```
Comment (elemanhillary) [inline config/index.js]: Kindly insert a new line

File: .sequelizerc (hunk 1 above)
Comment (Hervera) [reply .sequelizerc]: Ok, maybe that. have you corrected it?

**Pull request merged**